pom.xml

```diff
@@ -105,6 +105,7 @@
     <skipITs>${skipTests}</skipITs>
     <skipBuildImage>true</skipBuildImage>
     <snakeyaml.version>2.2</snakeyaml.version>
+    <prometheus.version>0.8.0</prometheus.version>
```

Comment:
Why not use ${prometheus.simpleclient.version}?

Reply:
done